markdown/events/2021/bytesize-4-github-contribution-basics.md

```diff
@@ -34,3 +34,82 @@ This will cover:
+
+So for example, there is a new feature; whenever there's a pull request open you can click on files changed and you can go through the files that have been changed in that pull request and click on insert a suggestion. [20:11](https://youtu.be/gTEXDXWf4hE?list=PL3xpfTVZLcNiSvvPWORbO32S1WDJqKp1e&t=1211)
+
+Whenever you click on the line of the code that would then open a text box, they can actually suggest a change that would make it possible later on for the person who opened the pull request to just click on accept that change. [20:17](https://youtu.be/gTEXDXWf4hE?list=PL3xpfTVZLcNiSvvPWORbO32S1WDJqKp1e&t=1217)
+
+For example for this type here it would be very easy to just type here and add such a suggestion fixing, and people don't have to write their own code again or go back to the editor because they can just click on this in the github interface and amend things. [20:46](https://youtu.be/gTEXDXWf4hE?list=PL3xpfTVZLcNiSvvPWORbO32S1WDJqKp1e&t=1246)
+
+Another good idea for effective code review is to describe the motivation and the purpose of the requested changes. For example if you have an idea or you find something particularly interesting, say you updated a mapper in an alignment pipeline, but the other person who opened the pull request might not know about then it would be a good idea to write that down in your code review and tell them about it. That's just one example but there are multiple others so you should always describe why you’re actually giving the feedback you are.
+
+It is a good idea to have a look at the tests, so for example we run certain linting tests which are checking the code for certain code structure and checks for markdown documentation as you see here. [21:45](https://youtu.be/gTEXDXWf4hE?list=PL3xpfTVZLcNiSvvPWORbO32S1WDJqKp1e&t=1303)
+
+So probably in the pull request that I opened yesterday, there is a is a failure in the markdown so the checks that we have in place actually failed, so you could check and click here on details and then find out what what's what's going on. [22:03](https://youtu.be/gTEXDXWf4hE?list=PL3xpfTVZLcNiSvvPWORbO32S1WDJqKp1e&t=1323)
+
+If you start a code review, or if you open a pull request, you can actually ask for code reviews. Github offers a nice feature on the right side where you can actually select who should review that pull request. Sometimes Github makes even suggestions, so suggested reviews are actually pre-selected by Github. That is typically done based on who contributed to that repository in the past, which is in many cases a good idea but not always the perfect idea because some people also are inactive now or might not actually have the time to work on that. So if you for example, you select one of the main developers on that pipeline as a review, but nothing happens for two days. There's also the possibility to ask in the request review channel on Slack, where you can indicate the pull request you just opened, and request another review.
```

Comment:
```suggestion
If you start a code review, or if you open a pull request, you can actually ask for code reviews. Github offers a nice feature on the right side where you can actually select who should review that pull request. Sometimes Github even makes suggestions, so suggested reviews are actually pre-selected by Github. That is typically done based on who contributed to that repository in the past, which is in many cases a good idea but not always the perfect idea because some people also are inactive now or might not actually have the time to work on that. So if you for example, you select one of the main developers on that pipeline as a review, but nothing happens for two days. There's also the possibility to ask in the request-review channel on Slack, where you can indicate the pull request you just opened, and request another review.
```